docs/installation.rst

```diff
@@ -10,7 +10,7 @@ The regions package requires the following packages:
 
 * Python 3.6 or later
 * `Numpy <https://numpy.org>`_ 1.16 or later
-* `Astropy <https://www.astropy.org>`__ 2.0 or later
+* `Astropy <https://www.astropy.org>`__ 3.2 or later
```

Comment:
This deserves a mention in the changelog.

Reply:
I updated the change log. Please take a look!